src/forms/inputs/dropdown-select.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { compose } from '../../utils'
+import { onOutsideClick, toggle, togglePropTypes } from '@launchpadlab/lp-hoc'
+import classnames from 'classnames'
+
+const propTypes = {
+  ...togglePropTypes('expanded'),
+  text: PropTypes.string,
+  selectedValues: PropTypes.arrayOf(PropTypes.string),
+  children: PropTypes.node,
+}
+
+const defaultProps = {
+  selectedValues: [],
+}
+
+// Wraps the `DropdownCheckboxGroup` component
+
+function DropdownSelect ({ 
+  children,
+  expanded, 
+  selectedValues, 
+  toggleExpanded, 
+}) {
+  return (
+    <div className="dropdown-select">
+      <div className="select-input" onClick={ toggleExpanded }>
+        <p>{ getLabel(selectedValues) }</p>
+      </div>
+      <div 
+        className={ classnames(
+          'options', 
+          'checkboxes', 
```

Comment:
This seems specific to the checkbox use case- to make it more extendable we could just include the `className` prop here and pass `'checkboxes'` in from `DropdownCheckboxGroup`